scene/2d/light_2d.cpp

```diff
@@ -425,6 +425,15 @@ real_t PointLight2D::get_texture_scale() const {
 	return _scale;
 }
 
+bool PointLight2D::_set(const StringName &p_name, const Variant &p_value) {
```

Comment:
This should be enclosed in `#ifndef DISABLE_DEPRECATED`.

Reply:
Done.
Also enclosed the function declaration in the header file. Seemed the right thing to do but let me know if I need to change it.

Reply:
Yes I meant to do it for both :thumbup: